api/src/main/java/org/openmrs/module/fgh/mpi/SnapshotEventProcessor.java

```diff
@@ -52,6 +58,15 @@ public SnapshotEventProcessor(int threadCount) {
 	
 	@Override
 	public void process(DatabaseEvent event) {
+		MpiContext mpiContext = null;
```

Comment:
Please add tests for changes you made to this class

Reply:
Did you see this comment?

Reply:
Done